README.md

```diff
@@ -86,15 +86,23 @@ aws_elb_request_count_sum{job="aws_elb",instance="",load_balancer_name="mylb",av
 aws_elb_request_count_sum{job="aws_elb",instance="",load_balancer_name="myotherlb",availability_zone="eu-west-1c",} 7.0
 ```
 
+### Tags
 If the `aws_tag_select` feature was used, an additional information metric will be exported for each AWS tagged resource matched by the resource type selection and tag selection (if specified), such as
 ```
 # HELP aws_resource_info AWS information available for resource
 # TYPE aws_resource_info gauge
 aws_resource_info{job="aws_elb",instance="",arn="arn:aws:elasticloadbalancing:eu-west-1:121212121212:loadbalancer/mylb",load_balancer_name="mylb",tag_Monitoring="enabled",tag_MyOtherKey="MyOtherValue",} 1.0
 ```
-
 All metrics are exported as gauges.
 
+The `aws_resource_info` can be used in conjunction with other metrics to expose tags using PromQL syntax like:
+```
+aws_elb_request_count_sum * on(load_balancer_name) aws_resource_info{tag_Env="prod", tag_Team="team-1", tag_MyOtherKey!="MyOtherValue"}
```

Comment:
Please split this across lines in the usual way, and the matchers here are unneeded.

Reply:
Do you mean splitting the expression itself over multiple lines? First PR ever to an OS project so not really sure what you mean with "the usual way".
And I included the matchers for clarity in the example, do you mean to have them all removed?

Reply:
See https://prometheus.io/docs/practices/rules/ for examples of formatting PromQL

Yes, remove them all. They'll more likely cause confusion.

Reply:
Ok will do. Personally, they would have helped by giving an example when working with cloudwatch-exporter, but I'd rather comply and merge something.